tests/components/mqtt/test_init.py

```diff
@@ -2909,7 +2775,8 @@ async def test_mqtt_subscribes_and_unsubscribes_in_chunks(
         task()
     await hass.async_block_till_done()
     # Make sure the debouncer finishes
-    await asyncio.sleep(0.2)
```

Comment:
In a future PR it would be nicer to patch/wrap debouncer so the test moves on as soon as it finishes

Reply:
Started https://github.com/home-assistant/core/pull/120373